tests/utils/dummyData.js

```diff
@@ -427,33 +427,33 @@ export const imageData = {
   file: "[object File]",
 };
 
-export const dummyEmptyPlioList = {
+global.dummyEmptyPlioList = {
   data: {
     count: 0,
     page_size: 5,
     results: [],
   },
 };
 
-export const dummyPlioList = {
+global.dummyPlioList = {
   data: {
     count: 5,
     page_size: 5,
     results: ["abc", "def", "ghi", "jkl", "mno"],
   },
 };
 
-export const dummyUniqueUserCountList = [1, 2, 3, 4, 5];
+global.dummyUniqueUserCountList = [1, 2, 3, 4, 5];
 
-export const dummyAccessToken = {
+global.dummyAccessToken = {
   access_token: "1234",
   expires_in: 86400,
   refresh_token: "4321",
   scope: "read write",
   token_type: "Bearer",
 };
 
-export let dummyItemResponses = [
+global.dummyItemResponses = [
```

Comment:
Just curious, how does the compiler know what type of variable this is? How does it know if it's a var, a let, or a const?

Reply:
given it is being added to the global object, it is likely global scope (hence, var) and not block scope!